include/openPMD/backend/Attributable.hpp

```diff
@@ -235,6 +236,9 @@ class AttributableImpl
     internal::SeriesInternal const & retrieveSeries() const;
     internal::SeriesInternal & retrieveSeries();
 
+    Iteration const & containingIteration() const;
+    Iteration & containingIteration();
```

Comment:
```suggestion
    /** Returns the corresponding Iteration
     *
     * Return the openPMD::iteration that this Attributable is contained in.
     * This walks up the linked parents until it finds the Iteration object.
     * Throws an error otherwise, e.g., for Series objects.
     * @{
     */
    Iteration const & containingIteration() const;
    Iteration & containingIteration();
    /** @} */
```